### PR TITLE
Add bullet package

### PR DESCRIPTION
**Pull request by ahcorde**
Package need it to compile tf2_bullet. https://github.com/ros2/geometry2/pull/205

I created the bullet.2.89.0.nupkg, where we should locate this pkg?

### CODE REVIEW

**Comment by ahcorde** [comment]
By default there is a `FindBullet.cmake` file inside CMake. In my case I'm using CMake 3.15. So I had to include [these](https://github.com/ros2/geometry2/pull/205/commits/d45d8d5e6f95e62b2fbb9720e53007e124bad7bc) lines in the CMakeLists.txt

File: package/bullet/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+This package is beta quality and designed for a specific application.
+At this time it is not recommended to distribute this package outside of the
+tutorial application which requires it.
+
+To build the package, follow these steps.
+
+You need Git, Visual Studio 2017 Command line tools, CMake, Chocolatey, and a text editor.
+
+- Clone or download https://github.com/bulletphysics/bullet3
+- Create a CMake build directory and run `cmake -G "Visual Studio 15 2017 Win64" PATH_TO_TINYXML_VENDOR -DBUILD_SHARED_LIBS=ON -DUSE_DOUBLE_PRECISION=ON -DBUILD_CPU_DEMOS=OFF -DBUILD_BULLET2_DEMOS=OFF -DBUILD_OPENGL3_DEMOS=OFF -DBUILD_UNIT_TESTS=OFF -DBUILD_EXTRAS=OFF`.
```
Comment (nuclearsandwich) [inline package/bullet/README.md]:
```suggestion
- Create a CMake build directory and run `cmake -G "Visual Studio 15 2017 Win64" PATH_TO_BULLET_SOURCE -DBUILD_SHARED_LIBS=ON -DUSE_DOUBLE_PRECISION=ON -DBUILD_CPU_DEMOS=OFF -DBUILD_BULLET2_DEMOS=OFF -DBUILD_OPENGL3_DEMOS=OFF -DBUILD_UNIT_TESTS=OFF -DBUILD_EXTRAS=OFF`.
```

File: package/bullet/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- Run `msbuild ALL_BUILD.sln /p:Configuration=Release`
+- Copy the Bullet*.lib and Bullet*.dll files to the package lib/ directory.
+- Run `msbuild ALL_BUILD.sln /p:Configuration=Debug`
```
Comment (nuclearsandwich) [inline package/bullet/README.md:15]: What resources need to be copied for debug?

File: package/bullet/bullet.nuspec [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <!-- == PACKAGE SPECIFIC SECTION == -->
+    <!-- This section is about this package, although id and version have ties back to the software -->
+    <!-- id is lowercase and if you want a good separator for words, use '-', not '.'. Dots are only acceptable as suffixes for certain types of packages, e.g. .install, .portable, .extension, .template -->
+    <!-- If the software is cross-platform, attempt to use the same id as the debian/rpm package(s) if possible. -->
+    <id>bullet</id>
+    <!-- version should MATCH as closely as possible with the underlying software -->
+    <!-- Is the version a prerelease of a version? https://docs.nuget.org/create/versioning#creating-prerelease-packages -->
+    <!-- Note that unstable versions like 0.0.1 can be considered a released version, but it's possible that one can release a 0.0.1-beta before you release a 0.0.1 version. If the version number is final, that is considered a released version and not a prerelease. -->
+    <version>2.89.0</version>
+    <packageSourceUrl>https://github.com/ros2/choco-packages/tree/latest/package/tinyxml2</packageSourceUrl>
```
Comment (nuclearsandwich) [inline package/bullet/bullet.nuspec]:
```suggestion
    <packageSourceUrl>https://github.com/ros2/choco-packages/tree/latest/package/bullet</packageSourceUrl>
```

**Review by nuclearsandwich — COMMENTED** [review]
Looks good so far. Added some suggestions for missed documentation and package fields.

**Comment by nuclearsandwich** [comment]
> I created the bullet.2.89.0.nupkg, where we should locate this pkg?

When this merges, we'll create a new release and upload this package along with the existing ones.

**Comment by ahcorde** [comment]
Thank you for the review @nuclearsandwich

**Comment by ahcorde** [comment]
@nuclearsandwich did you have the chance to have a look?

File: package/bullet/README.md (hunk 2 above)
Comment (nuclearsandwich) [reply package/bullet/README.md:15]: Resolved by 6fed593

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**